plugins/aws/lex/lexAudioLogsEncrypted.js

```diff
@@ -0,0 +1,148 @@
+            
+        if (!listBots) return callback(null, results, source);
+
+        if (listBots.err || !listBots.data) {
+            helpers.addResult(results, 3,
+                'Unable to query for  Lex bots: ' + helpers.addError(listBots));
+            return callback(null, results, source);
+        }
+
+        if (!listBots.data.length) {
+            helpers.addResult(results, 0, 'No  Lex bots found');
+            return callback(null, results, source);
+        }
+
+        var listKeys = helpers.addSource(cache, source,
+            ['kms', 'listKeys', region]);
+
+        if (!listKeys || listKeys.err || !listKeys.data) {
+            helpers.addResult(results, 3,
+                `Unable to list KMS keys: ${helpers.addError(listKeys)}`, region);
+            return callback(null, results, source);
+        }
+
+        async.each(listBots.data, function(bot, cb){
+            if (!bot.botId) return cb();
+
+            var listBotAliases = helpers.addSource(cache, source,
+                ['lexmodelsv2', 'listBotAliases', region, bot.botId]);
+                // console.log(listBotAliases.data);
+
+            if (!listBotAliases || listBotAliases.err || !listBotAliases.data) {
+                helpers.addResult(results, 3,
+                    'Unable to query for Lex bot aliases: ' + bot.botId + ': ' + helpers.addError(listBotAliases), region);
+                return cb();
+            }
+
+            if (!listBotAliases.data.botAliasSummaries || !listBotAliases.data.botAliasSummaries.length) {
+                helpers.addResult(results, 3,
+                    'Unable to query for Lex bot aliases descriptions: '  + helpers.addError(listBotAliases), region);
+                return cb();
+            }
+
+            for (let alias of listBotAliases.data.botAliasSummaries) {
+           
+                var resource = `arn:${awsOrGov}:lex:${region}:${accountId}:bot/${alias.botAliasId}`;
+
+                var describeBotAlias = helpers.addSource(cache, source,
+                    ['lexmodelsv2', 'describeBotAlias', region, alias.botAliasId]);
+                    // console.log(JSON.stringify(describeBotAlias , null, 2));
+
+                if (!describeBotAlias ||
+                    describeBotAlias.err ||
+                    !describeBotAlias.data) {
+                    helpers.addResult(results, 3,
+                        'Unable to get Lex bot aliases description: ' + alias.botAliasId + ': ' + helpers.addError(describeBotAlias), region, resource);
+                    continue;
+                }
+
+                if (!describeBotAlias.data.conversationLogSettings) {
+                    helpers.addResult(results, 0,
+                        'Unable to get Lex conversation log settings: ' + alias.botAliasId + ': ' + helpers.addError(describeBotAlias), region, resource);
+                    continue;
+                }
+
+                if (describeBotAlias.data.conversationLogSettings  && 
```

Comment:
Add login to loop describeBotAlias.data.conversationLogSettings.audioLogSettings